van-gogh-second-page/dalite/style.css

```diff
@@ -0,0 +1,84 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-weight: 500;
+}
+body{
+    background-color: #F497AF;
+    font-family: Gotham Rounded SSm,Arial,sans-serif;
+}
+.navigation{
+    list-style-type: none;
+    margin: 30px 45px; 
+    padding: 0;
+    overflow: hidden;
+    cursor: pointer;
+}
+ul{
+    display: flex;
+    
+}
+li a{
+    display: block;
+    color: black;
+    text-align: center;
+    margin: 30px;
+    text-decoration: none;
+}
+.logo{
+    color: white;
+    background-color: black;
+    height: 70px;
+    width: 70px;
+    margin: 0 30px;
+    text-align: end;padding: 10px 10px 0px 0px;
+    font-size: 14px;
+    position: absolute;
+    top: 70px;
+    right: 20px;
+    cursor: pointer;
+
+}
+span{
```

Comment:
It selects every single span on the page, I think it's better to update the specific span we are interested in, cause for the other spans we might want another behaviour